fields/types/cloudinaryimage/CloudinaryImageField.js

```diff
@@ -301,7 +301,7 @@ module.exports = Field.create({
 			fieldClassName = 'field-ui';
 
 		if (hasImage) {
-			fieldClassName += ' has-image';
+			fieldClassName += 'has-image';
```

Comment:
This will break, css classes are space-delimited and we're building a string. This will end up with `'field-uihas-image'` instead of `'field-ui has-image'`.